07.ls_object/ls_object.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require_relative 'entry'
+require_relative 'display'
+require 'optparse'
+
+def main
+  options = parse_options
+  entry_names = if options[:a]
+                  Dir.entries('.').sort
+                else
+                  Dir.glob('*')
+                end
+  entries = entry_names.map { |entry_name| Entry.new(entry_name) }
+  display_entries = options[:r] ? entries.reverse : entries
```

Comment:
```suggestion
  sorted_entries = options[:r] ? entries.reverse : entries
```

右辺でやっていることは並び替えなので、↑みたいな名前の方がコードを読んだ人にとっては自然かも

Reply:
Displayクラスのdisplay_entriesメソッドと名前が被ってしまうのも紛らわしい点ですね

Reply:
修正しました。